src/components/AvatarWrapper/AvatarWrapper.vue

```diff
@@ -134,7 +139,7 @@ export default {
 			if (this.isDeletedUser) {
 				return 'X'
 			}
-			const customName = this.name !== t('spreed', 'Guest') ? this.name : '?'
+			const customName = this.name && this.name !== t('spreed', 'Guest') ? this.name : '?'
```

Comment:
Is this an effective change? What is the difference and is it needed to do this on stable26 and others?

Reply:
`this.name &&` check that name is non-empty, like ""

Reply:
ah, then I guess it should be backported